### PR TITLE
feat: reverse address to domian

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-249

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/encryption/src/encryption/Decryption.ts [hunk 1]
```diff
@@ -116,7 +116,6 @@ async function* v37ECDHE(
                 encryptedPostKey: encryption.ownersAESKeyEncrypted.val,
                 postKeyIV: iv,
             }
-            debugger
```
Comment (guanbinrui) [inline packages/encryption/src/encryption/Decryption.ts]: !

Comment (nuanyang233) [reply packages/encryption/src/encryption/Decryption.ts]: No debugger eslint

**Comment by wenluomask** [comment]
Checked.

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
LGTM

File: packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx [hunk 2]
```diff
@@ -301,7 +301,7 @@ export function CollectionList({
                                 className={classes.button}
                                 variant="outlined"
                                 size="small">
-                                {formatEthereumAddress(addressName.label, 5)}
+                                <ReverseAddress address={addressName.resolvedAddress} addressSize={5} />
```
Comment (septs) [inline packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx:304]: suggest, `5` value by address size default, don't repeat yourself.

rename `addressSize` to `size`.

Comment (septs) [reply packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx:304]: suggest, rename the component name to `ReversedAddress`

the is react component, should be used when done.

e.g: `formatAddress` as component, `FormattedAddress`

Comment (nuanyang233) [reply packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx:304]: resolved in #5914

**Pull request merged**